tests/data_validation/great_expectations/expectations/verify_current_week_article.json

```diff
@@ -0,0 +1,52 @@
+{
+  "data_asset_type": null,
+  "expectation_suite_name": "verify_current_week_article",
+  "expectations": [
+    {
+      "expectation_type": "expect_column_values_to_be_null",
+      "kwargs": {
+        "column": "newsletter_id",
+        "mostly": 0.1
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_value_lengths_to_be_between",
+      "kwargs": {
+        "column": "description",
+        "condition_parser": "great_expectations__experimental__",
+        "min_value": 10,
+        "mostly": 0.1,
+        "row_condition": "col(\"category_id\")==3"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_value_lengths_to_be_between",
```

Comment:
This expectation doesn't filter the data with column "newsletter_id"= "null". @akkuldn Please update the expectation.